server/go/.golangci.yml

```diff
@@ -55,7 +55,9 @@ linters-settings:
   gci:
     sections:
       - standard
-      - prefix(github.com/mlrun/mlrun)
+      - prefix(github.com/mlrun/framework)
+      - prefix(github.com/mlrun/proto)
+      - prefix(github.com/mlrun/services)
```

Comment:
what's the reason to separate the import groups of internal mlrun packages?

Reply:
just to be able to differentiate between the components as they suppose to be orthogonal